jerry-core/parser/js/js-parser-expr.c

```diff
@@ -1809,6 +1775,13 @@ parser_process_unary_expression (parser_context_t *context_p, /**< context */
           context_p->last_cbc_opcode = PARSER_CBC_UNAVAILABLE;
           parser_emit_cbc_literal_from_token (context_p, CBC_PUSH_PROP_THIS_LITERAL);
         }
+#if ENABLED (JERRY_ES2015)
+        else if (context_p->last_cbc_opcode == PARSER_TO_EXT_OPCODE (CBC_EXT_PUSH_SUPER))
```

Comment:
I think the flush opcode should have a check which disallows flushing this opcode.